server.js

```diff
@@ -0,0 +1,16 @@
+var express = require('express');
+var bodyParser = require('body-parser');
+var curses = require('./curses.json').curses;
+
+var app = express();
+var port = process.env.PORT || 3000;
+
+app.use(express.static(__dirname + '/'));
```

Comment:
I'll update the PR to include `path`. I'm not able to deploy a Heroku instance of your project, only my fork. I suggest enabling Automatic Deployments so you don't have to juggle multiple remotes.